src/webcomponents/helperfunctions/jsonToCol.ts

```diff
@@ -0,0 +1,40 @@
+import { JsonToCol, LanguagesInput } from "../../@types/interfaces";
+
+const mapLangToSheetColumns = (languages: LanguagesInput) => {
+  let columnTitle: Array<object> = Object.keys(
+    languages
+  ).map((title: string) => ({ type: "text", title: `${title}`, width: 150 }));
+  return columnTitle;
+};
+
+const setKeys = (languages: LanguagesInput): Array<string> => {
+  const keyValue = new Set();
+  const reducer = (accumulator: object, currentValue: object) =>
```

Comment:
try to use accumulator as `Set` (please check some examples on internet regarding "reduce to Set")